docs/index.md

```diff
@@ -1,22 +1,30 @@
 # MCUboot
 
+MCUboot is a secure bootloader for 32-bit MCUs.
+
 ## Overview
 
-MCUboot is a secure bootloader for 32-bit MCUs.   The goal of MCUboot is to
-define a common infrastructure for the bootloader, system flash layout on
-microcontroller systems, and to provide a secure bootloader that enables
-easy software upgrade.
+The goal of MCUboot is to define a common infrastructure for the bootloader,
+system flash layout on microcontroller systems, and to provide a secure
+bootloader that enables easy software upgrade.
+
+MCUboot is operating-system and hardware independent, and relies on
+hardware porting layers from the operating system it works with. Currently
+MCUboot works with the following operating systems:
+- [Zephyr](https://www.zephyrproject.org/)
+- [Apache Mynewt](https://mynewt.apache.org/)
+- [Apache NuttX](https://nuttx.apache.org/)
+- [RIOT](https://www.riot-os.org/)
+- [Mbed OS](https://os.mbed.com/)
+- [Espressif IDF](readme-espressif.md)
```

Comment:
Again, have Cypress/Infineon in the list.

Reply:
fixed